e2e/tests/trustless_relays.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Trustless Relays
```

Comment:
Can you add a command in the Makefile to run a `make e2e_relay_test`

Reply:
Thank you for the review. Updated.

Reply:
Used `test_e2e_relay` as the new target, as it seemed more consistent. Please advise if this needs to be changed.